README.md

```diff
@@ -0,0 +1,134 @@
+It uses a [TPM](https://en.wikipedia.org/wiki/Trusted_Platform_Module) — such as Intel SGX or ARM TrustZone — ensuring reliable and trustworthy execution of the platform without the need for public verification of all computations (in comparison to accumulated zero-knowledge proofs, TPMs allow the generation of more compact and less computationally expensive proofs, thereby scaling much better and consuming less power).
+In Erdstall, the TEE operator periodically generates checkpoints, also called balance proofs, that can be used to withdraw funds on-chain.
+
+**Benefits of Erdstall**&emsp;<!--
+-->In addition to scaling payments as a layer-2 platform, Erdstall also specifically aims to be the secure asset management layer for non-custodial off-chain marketplaces.
+A marketplace building on top of Erdstall does not have to manage any funds on its own, it only has to match the trading parties, while earning a flat fee per facilitated trade.
+This makes the marketplace more trustworthy, as a cyber-attack on the marketplace cannot result in lost funds.
+As a result, Erdstall allows users to enjoy the security, control, and trust of non-custodial trading, combined with the low cost of off-chain transacting.
+Meanwhile, market operators benefit from fast throughput and low operation costs, innate user-trust, and fast and cheap deposits and withdrawals via Erdstall.
+Another benefit is that content creators can natively mint NFTs off-chain, and the cost of on-chain token creation only happens when the NFT is withdrawn by a user into the Ethereum mainnet.
+Because our protocol is very light-weight, it is easy to integrate into existing solutions.
+
+**Scope of this SDK**&emsp;<!--
+-->This SDK is designed to offer the complete client-side functionality of Erdstall.
+It natively integrates with any ethers.js provider, such as Metamask, as its wallet manager.
+The SDK offers full control, but also packages the most important functionality into an easy-to-use API.
+The SDK is designed such that code snippets that use Erdstall can be written generically to be reused within any site using Erdstall.
+That way, common web components, such as payment/donation buttons, can easily be reused on your own homepage.
+
+## Getting started
+
+To get your Erdstall app started, there are a few steps you need to take.
+
+**Importing Erdstall**&emsp;<!--
+-->First, import the SDK's main source directory:
+
+```ts
+import * as SDK from "erdstall-ts-sdk/src"
+```
+
+**Creating the client object**&emsp;<!--
+-->The SDK is straight-forward to use: everything you need is neatly bundled in the `SDK.Erdstall` interface, which you can instantiate via `SDK.NewClient()`, as follows:
+
+```ts
+// Initialize these fields with values received from your ethers or web3 setup.
+var ethAccount: {address: SDK.ledger.Address, signer: ethers.Signer};
+// Create client object.
+let Erdstall = SDK.NewClient(
```

Comment:
```suggestion
let client = SDK.NewClient(
```
or `erdclient`.

Reply:
In particular, local variables of class instances usually don't start with a capital letter.